js/interface.js

```diff
@@ -390,6 +392,42 @@ function trimColumns(columns) {
   });
 }
 
+function getEmptyColumns(columns, entries) {
+  var emptyColumns = _.filter(columns, function(column) {
+    return emptyColumnNameRegex.test(column);
+  });
+
+  if (emptyColumns.length) {
```

Comment:
Avoid the `if` indentation with this.

```suggestion
  if (!emptyColumns.length) {
    return emptyColumns;
  }
```

Reply:
Fixed.